test/conformance.go

```diff
@@ -48,6 +48,7 @@ const (
 	PizzaPlanet2        = "pizzaplanetv2"
 	Protocols           = "protocols"
 	Readiness           = "readiness"
+	ReadinessPort       = "readinessport"
```

Comment:
Any reason why we reverted to having two separate test images?

Reply:
Looks like this line was added back when it shouldn't have been. Since it looks like the edits are in the existing image